### PR TITLE
Improve CLI

### PR DESCRIPTION
**Pull request by sir-marc**

### CODE REVIEW

File: cli/lib/files.js [hunk 1]
```diff
@@ -9,6 +9,11 @@ const createFolderPath = (dirname) => {
   fs.mkdirSync(dirname, { recursive: true });
 };
 
+export const getFolderPath = (filePath) => path.dirname(filePath);
+export const getFilename = (filePath) => path.basename(filePath);
+export const getFilenameWithoutExtension = (filename) =>
+  filename.split(".").slice(0, -1).join(".");
```
Comment (StuckiSimon) [inline cli/lib/files.js:15]: mhhhmmmm, string magic

File: cli/lib/files.js [hunk 2]
```diff
@@ -20,3 +25,7 @@ export const createFile = (filePath, content) => {
   createBaseFolderPathForFile(filePath);
   fs.writeFileSync(filePath, content);
 };
+
+export const rmDir = (path) => {
+  fs.rmSync(path, { recursive: true, force: true });
```
Comment (StuckiSimon) [inline cli/lib/files.js:30]: force? 😅

**Review by StuckiSimon — APPROVED** [review]
Looks nice 👍 
Btw: for config files there is a nice library (which e.g. prettier uses) https://github.com/davidtheclark/cosmiconfig

File: cli/lib/files.js (hunk 2 above)
Comment (sir-marc) [reply cli/lib/files.js:30]: For once i acted according to the docs:
`node:47507) [DEP0147] DeprecationWarning: In future versions of Node.js, fs.rmdir(path, { recursive: true }) will throw if path does not exist or is a file. Use fs.rm(path, { recursive: true, force: true }) instead`

**Pull request merged**